Dockerfile

```diff
@@ -25,6 +25,9 @@ RUN groupadd -r geodata -g 2500 && \
     # create mountpoint folders with geodata ownership
     install -o geodata -g geodata -d /var/lib/sphinxsearch/data/index/ && \
     install -o geodata -p geodata -d /var/lib/sphinxsearch/data/index_efs/ && \
+    # redirect logs to stdout
```

Comment:
@ltshb found this solution to redirect the application logs of searchd (query.log and searchd.log) to stdout

Reply:
@ltclm  Are you sure then that logs are not sent twice ?